src/java/org/apache/cassandra/db/lifecycle/Tracker.java

```diff
@@ -269,13 +271,33 @@ public void reset(Memtable memtable)
                           SSTableIntervalTree.empty()));
     }
 
-    public Throwable dropSSTablesIfInvalid(Throwable accumulate)
+    public Throwable dropOrUnloadSSTablesIfInvalid(String message, @Nullable Throwable accumulate)
     {
         if (!isDummy() && !cfstore.isValid())
-            accumulate = dropSSTables(accumulate);
+            accumulate = dropOrUnloadSSTables(cfstore.status(), message, accumulate);
         return accumulate;
     }
 
+    /**
+     * Drop sstables if cfs is invalidated with drop data; otherwise unload sstables
```

Comment:
`cfs` is `cfstore` in this class, thus I suggest
```suggestion
     * Drop sstables if cfstore is invalidated with drop data; otherwise unload sstables
```